src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -18,29 +17,34 @@ public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a person. "
+            + "Parameters: "
+            + PREFIX_STUDENTNUMBER + "STUDENT NUMBER"
```

Comment:
Newline could be added for better readability.  :blush:
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a person.\n"
            + "Parameters: "
            + PREFIX_STUDENTNUMBER + "STUDENT NUMBER\n"
```